### DISCUSSION TITLE
Unable to Scan with -turl Option in Docker Image

### DISCUSSION THREAD

**Discussion by 1205747671** [projectdiscovery/nuclei#4682] [Q&A] [answered]
I’m encountering an issue when running the Nuclei Docker image (projectdiscovery/nuclei:latest). When I specify a template URL with the -turl option, the scan fails with the following error message:

`[FTL] Could not run nuclei: could not load templates from config: Remote template URL host (raw.githubusercontent.com) is not present in the `remote-template-domain` list in nuclei config
`
Here’s the command I’m using:

`docker run --rm -v G:\tmp\:/nuclei/outputs projectdiscovery/nuclei -u https://example.com/ -turl https://raw.githubusercontent.com/projectdiscovery/nuclei-templates/master/cves/2021/CVE-2021-42258.yaml -c 50 -je /nuclei/outputs/nuclei_example.com_aaa.json
`

It seems like the Docker image doesn’t include raw.githubusercontent.com in the remote-template-domain list in the Nuclei config. This prevents me from using remote templates hosted on GitHub.

What should I do?

**Comment by dogancanbakir** [comment]
This is the only trusted template domain https://github.com/projectdiscovery/nuclei/blob/b9e2665e9e4f341a16081ce07808a5d4cc28f4f2/pkg/catalog/loader/loader.go#L38. You can either create a config file that contains remote template domain you want inside the container or mount it to the docker.

**Comment by olearycrew** [comment, marked as answer]
Alternatively I think you just want to pass the URL to a template - which you can just do with `-t` - as @dogancanbakir mentioned that is not what `-turl` is for.
